### ISSUE TITLE
Post with large binary file throws net::ERR_SPDY_STREAM_CLOSED error.

### ISSUE THREAD

**Issue by ipbharaj** [ThreeMammals/Ocelot#762]
## Expected Behavior 
I have a REST API which accepts post call and uploads the binary file into data base, which is hosted in IIS
I set up Ocelot API Gateway with basic configurations: 
```json
"DownstreamPathTemplate": "/{everything}",
            "DownstreamScheme": "http",
            "DownstreamHostAndPorts": [
              {
                "Host": "localhost1",
                "Port": 80
              }
                ],
            "UpstreamPathTemplate": "/{everything}"
```
It should complete my request as it is doing if I call the Upload API directly.

## Actual Behavior 

When calling through Api gate way it gives net::ERR_SPDY_STREAM_CLOSED error if the file size is anything more than 1 MB

## Specifications

  - Version:
    Ocelot Version="12.0.1"
netcoreapp2.1
  - Platform:
Linux docker container

  - Subsystem:

 - Configuration:
```json
{
    "ReRoutes": [
        {
            "DownstreamPathTemplate": "/{everything}",
            "DownstreamScheme": "http",
            "DownstreamHostAndPorts": [
              {
                "Host": "localhost1",
                "Port": 80
              }
                ],
            "UpstreamPathTemplate": "/{everything}"
        }
    ],
    "GlobalConfiguration": {
        "DangerousAcceptAnyServerCertificateValidator": false
    }
}
```

**Comment by tmapfumo** [comment]
Any news on this issue, experiencing the same problem on a Linux hosted API gateway

**Comment by raman-m** [comment]
It should be fixed by #1724.
So, pumping large files should work fine in [Nov-December'23](https://github.com/ThreeMammals/Ocelot/milestone/2) release.
Please watch for this release in a couple of weeks.

If it doesn't help than it should be fixed by #1824 definitely. Watch for [Annual 2023](https://github.com/ThreeMammals/Ocelot/milestone/3) release...
@ipbharaj I don't recommend you to host Ocelot in IIS. We are not responsible for your unstable IIS environment. Please use self-hosted deployments.
